connectors/sources/sharepoint_online.py

```diff
@@ -793,6 +793,14 @@ def get_default_configuration(cls):
                 "type": "bool",
                 "value": False,
             },
+            "use_document_level_security": {
+                "display": "toggle",
+                "label": "Use document level security",
+                "order": 7,
+                "tooltip": "This will add access control data to content documents and will also sync identities with their corresponding access control to a separate index .search-acl-filter-index-name",
```

Comment:
I feel my proposal is too vague. My point is that the tooltip is supposed to be displayed in the UI, and it's purpose is to tell customers that enabling this will apply the access control in the remote source to all the data in ES. We don't need to expose details like `.search-acl-filter-index-name`

Reply:
Maybe @leemthompo can help? 🙏

Reply:
I agree with Chenhui's point that the tooltip doesn't need to go into huge detail.

Something like the following might work:

_Document level security ensures identities and permissions set in Sharepoint Online are maintained in Elasticsearch. This enables you to restrict and personalize read-access users have to documents in this index.
Access control syncs ensure this metadata is kept up to date in your Elasticsearch documents._